src/ucs/sys/event_set.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (C) Hiroyuki Sato. 2019. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_EVENT_SET_H
+#define UCS_EVENT_SET_H
+
+#include <ucs/type/status.h>
+
+/**
+ * ucs_sys_event_set structure used in ucs_event_set_XXX functions.
+ *
+ */
+typedef struct ucs_sys_event_set ucs_sys_event_set_t;
+
+
+/**
+ * ucs_event_set_handler call this handler for notifying event
+ *
+ */
+typedef ucs_status_t (*event_set_handler_t)(int nready, void *event);
```

Comment:
1. what is the point to pass void* to the callback? how would the user know if it's kqueue or epoll event format?
1. need to define proper name, eg ucs_event_set_handler_t

Reply:
@hiroyuki-sato the API should device some platform-independent wrapper for event data, like you did for flags. and pass it as the argument instead of void* - since the goal of this PR is to have platform-independent API. this way also the unit test will not have to include epoll.h

Reply:
Removed `sys/epoll.h`